compiler/ballerina-lang/src/main/java/io/ballerina/projects/internal/model/PackageJson.java

```diff
@@ -162,7 +162,7 @@ public void setPlatformDependencies(List<PlatformLibrary> platformDependencies)
         this.platformDependencies = platformDependencies;
     }
 
-    public boolean isTemplate() {
+    public boolean getTemplate() {
```

Comment:
Any particular reason we are changing this to `getTemplate()`? Since its a boolean, wouldnt `isTemplate()` make more sense?